hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -507,6 +508,13 @@ public static SchemaProviderWithPostProcessor wrapSchemaProviderWithPostProcesso
         Option.ofNullable(createSchemaPostProcessor(schemaPostProcessorClass, cfg, jssc)));
   }
 
+  public static SchemaProvider schemaProviderWithKafkaOffsetPostProcessorIfNeeded(SchemaProvider provider, TypedProperties cfg, JavaSparkContext jssc) {
```

Comment:
getSchemaProviderForKafkaSource should suffice. lets add java docs to clarify what this method is doing. Its too lengthy to add KafkaPostProcessor in the naming